app/uk/gov/hmrc/tai/service/helper/TaxCodeIncomeHelper.scala

```diff
@@ -14,45 +14,29 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.tai.repositories.deprecated
+package uk.gov.hmrc.tai.service.helper
 
 import com.google.inject.{Inject, Singleton}
+import play.api.Logging
 import uk.gov.hmrc.domain.Nino
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.tai.connectors.TaxAccountConnector
-import uk.gov.hmrc.tai.model.domain.formatters.income.{TaxAccountIncomeHodFormatters, TaxCodeIncomeHodFormatters}
 import uk.gov.hmrc.tai.model.domain.formatters.IabdDetails
+import uk.gov.hmrc.tai.model.domain.formatters.income.TaxCodeIncomeHodFormatters
```

Comment:
This is really a service, not a helper. It's in the service package and it is a class. Generally helpers are objects not classes. Can we rename this to TaxCodeIncomeService?